ffi/util.lua

```diff
@@ -60,10 +60,27 @@ int WideCharToMultiByte(
 );
 ]]
 
+local getlibprefix = function()
```

Comment:
So that didn't turn out as a oneliner. `os.popen(...) or ""` was throwing errors, and as I found in other parts of the code that `os.popen` has to be taken care of as a `fopen`.